cmd/bosun/expr/funcs.go

```diff
@@ -315,6 +316,12 @@ var builtins = map[string]parse.Func{
 		Tags:   tagFirst,
 		F:      TimeDelta,
 	},
+	"tail": {
+		Args:   []models.FuncType{models.TypeSeriesSet, models.TypeScalar},
```

Comment:
You might consider a numberSet instead of a scalar and then use something like `match`. This way your argument to tail could the be the result of another reduction function. Want to convert more functions to be like this